Author: Ansgar Burchardt <ansgar@debian.org>
Description: Do not call MPI_Init() from within library; applications are responsible for this. (Closes: #736842)
--- a/Ipopt/src/Algorithm/LinearSolvers/IpMumpsSolverInterface.cpp
+++ b/Ipopt/src/Algorithm/LinearSolvers/IpMumpsSolverInterface.cpp
@@ -54,11 +54,11 @@
                    dbg_verbosity);
     //initialize mumps
     DMUMPS_STRUC_C* mumps_ = new DMUMPS_STRUC_C;
-    int argc=1;
-    char ** argv = 0;
-    int myid;
-    MPI_Init(&argc, &argv);
-    MPI_Comm_rank(MPI_COMM_WORLD, &myid);
+    // int argc=1;
+    // char ** argv = 0;
+    // int myid;
+    // MPI_Init(&argc, &argv);
+    // MPI_Comm_rank(MPI_COMM_WORLD, &myid);
     mumps_->n = 0;
     mumps_->nz = 0;
     mumps_->a = NULL;
@@ -84,7 +84,7 @@
     DMUMPS_STRUC_C* mumps_ = (DMUMPS_STRUC_C*)mumps_ptr_;
     mumps_->job = -2; //terminate mumps
     dmumps_c(mumps_);
-    MPI_Finalize();
+    // MPI_Finalize();
     delete [] mumps_->a;
     delete mumps_;
   }
